src/Grpc.AspNetCore.Server/Internal/GrpcServiceBinder.cs

```diff
@@ -178,7 +178,7 @@ private GrpcContentTypeConstraint()
 
```

Comment:
The `AddMethod` methods are causing errors in VS:
```
Severity	Code	Description	Project	File	Line	Suppression State
Error	CS8634	The type 'TResponse' cannot be used as type parameter 'TResponse' in the generic type or method 'ServerCallHandlerFactory<TService>.CreateUnary<TRequest, TResponse>(Method<TRequest, TResponse>, UnaryServerMethod<TService, TRequest, TResponse>)'. Nullability of type argument 'TResponse' doesn't match 'class' constraint.	Grpc.AspNetCore.Server	C:\gh\grpc-dotnet\src\Grpc.AspNetCore.Server\Internal\GrpcServiceBinder.cs	86	Active
```

I think you need to add more constraints.

Reply:
It builds on my computer and the build server. What is special to your computer that fails?

Reply:
Yea it always built from the command line. It seems like my VS was out of date, updating to 16.1 preview 3 fixed it.